ams/dashboard/src/views/tables/components/Transactions.vue

```diff
@@ -123,10 +123,18 @@ async function getTableInfo() {
     const fcData: ILineChartOriginalData = {}
     list.forEach((p: TransactionItem) => {
       // Assume that the time will not conflict and use the time as the unique key without formatting it.
-      const { summary } = p
+      const { summary, recordsSummaryForChart, filesSummaryForChart, commitTime } = p
       const { 'total-records': totalRecords, 'total-equality-deletes': totalEqualityDeletes, 'total-position-deletes': totalPositionDeletes, 'total-data-files': totalDataFiles, 'total-delete-files': totalDeleteFiles } = summary
```

Comment:
If I understand correctly, `summary` should no longer be in use for charting, so we can simplify the code, and only support getting data from `recordsSummaryForChart` and `filesSummaryForChart` for charting. 😄 @minteliuwm